.github/workflows/publish-bicep.yaml

```diff
@@ -26,47 +26,13 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 10
     env:
-      GOVER: '^1.21'
+      GOVER: "^1.21"
```

Comment:
was this changed needed?

Reply:
We should be able to remove this, we don't use Go here.